mitype/commandline.py

```diff
@@ -155,3 +170,30 @@ def load_based_on_difficulty(difficulty_level=random.randrange(1, 6)):
 
     print("Select a difficulty level in range [1,5]")
     sys.exit(2)
+
+
+def show_history(N):
+    N = abs(int(N))
+    history_file = 'mitype_history.csv'
+    history_path = os.path.join(os.path.expanduser('~'), history_file)
+
+    if os.path.exists(history_path):
+        with open(history_path, 'r') as file:
+            history_reader = csv.reader(file)
+            next(history_reader)
+
+            data = list(history_reader)
+            no_of_records = len(data)
+
+            k = no_of_records if N == -1 or N >= no_of_records else N
+            print("Last " + str(k) + " records: ")
```

Comment:
`print("Last",k,"records")` would work :)

Reply:
This work too right? 🤔 Is it not supported in Python2 or is it less efficient? It's the same right?

Reply:
This does work, but for the sake of consistency.
Also it looks better, than converting it to string first!